catalog/lib/widgets/app_checkbox.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:catalog/catalog.dart';
+import 'package:flutter/material.dart';
+
+typedef AppCheckboxItems<T> = ({bool value, String label});
+
+class AppCheckbox<T> extends StatefulWidget {
+  final bool initialValue;
+  final List<AppCheckboxItems> items;
+  final void Function(bool?) onPressed;
+
+  const AppCheckbox({
+    required this.onPressed,
+    required this.items,
+    required this.initialValue,
```

Comment:
Add an axis parameter so we can change it to horizontal and vertical orientation